dev.zed.Zed.yaml

```
@@ -0,0 +1,91 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: Zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
+  - --device=dri
+
+  # Allow access to the user's home directory for reading and writing files
+  - --filesystem=home
+
+  # Allow shared memory access for inter-process communication
+  - --share=ipc
+
+  # Enable network access
+  - --share=network
+
+  # Provide access to X11 display server as a fallback
+  - --socket=fallback-x11
+
+  # Enable access to PulseAudio sound server
+  - --socket=pulseaudio
+
+  # Enable access to Wayland display server
+  - --socket=wayland
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+modules:
+  - name: zed
+    buildsystem: simple
+    build-commands:
+      # Fetch dependencies
+      - cargo fetch --manifest-path Cargo.toml --offline --verbose
+
+      # Build Zed with parallel jobs
+      - cargo build --release --locked --all-features --jobs $(nproc)
```

```suggestion
      - cargo build --release --locked --all-features
```

don't use `nproc`. If you really want to add it (cargo does it by itself) use `$FLATPAK_BUILDER_N_JOBS` which honour the flatpak-builder `--jobs` argument.

---

No need to use Cargo build anymore if using official Github release